docs/user/api/v3.rst

```diff
@@ -561,6 +572,9 @@ Version detail
                           Allowed values are ``last_build`` and ``last_build.config``.
                           Multiple fields can be passed separated by commas.
 
+    .. note::
+
+       Privacy levels are only available in :doc:`Read the Docs for Business </commercial/index>`.
```

Comment:
I don't like to have to replicate this note on every serializer where these fields appear. However, I don't have a better suggestion. Maybe @benjaminp has already dealt with this case and have a good suggestion here?